src/IW.hs

```diff
@@ -20,6 +21,9 @@ mkAppEnv Config{..} = do
     -- IO configuration
     envDbPool <- initialisePool cDbCredentials
 
+    -- Http manager configuration
+    envManager <- newManager defaultManagerSettings
```

Comment:
You should use `newTlsManager` function from `http-client-tls` package here to be able to download from HTTPS urls

Reply:
@chshersh Got it!